daisy/step_wait_for_instances_signal.go

```diff
@@ -178,6 +192,61 @@ func waitForSerialOutput(s *Step, project, zone, name string, so *SerialOutput,
 	}
 }
 
+func waitForGuestAttribute(s *Step, project, zone, name string, ga *GuestAttribute, interval time.Duration) DError {
+	w := s.w
+	msg := fmt.Sprintf("Instance %q: watching for key %s", name, ga.KeyName)
+	if ga.SuccessValue != "" {
+		msg += fmt.Sprintf(", SuccessValue: %q", ga.SuccessValue)
+	}
+	w.LogStepInfo(s.name, "WaitForInstancesSignal", msg+".")
+	var errs int
+	tick := time.Tick(interval)
```

Comment:
There are limitations how often guest attributes can be read (10 reads per minute and 3 reads per second, otherwise guest attributes could get deleted): https://cloud.google.com/compute/docs/metadata/manage-guest-attributes#when_to_use_guest_attributes

`WaitForInstanceSignal` should verify that interval provided won't exceed these limits.

Reply:
the deletion caveat is solely for writes; in this code, the worst case is we receive a throttle exception. will update the tick interval however

Reply:
default interval is already 10s

Reply:
Interval is configurable. What stops a user from setting a 50ms interval?

Reply:
my assumption was that users can set invalid configurations. if it's more normal to prevent this, i could add a limit to interval to use user-provided or 10s, whichever is greater

Reply:
added mininim interval of 6s